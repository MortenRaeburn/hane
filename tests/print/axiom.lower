Axiom ax : forall[Set] ('0).
Print ax.
