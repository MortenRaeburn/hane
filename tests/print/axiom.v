Axiom ax : forall A : Set, A.
Print ax.
