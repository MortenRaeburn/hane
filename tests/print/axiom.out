Axiom ax : forall[Set] ('0).
